libdrgn/linux_kernel_object_find.inc.strswitch

```diff
@@ -33,6 +33,10 @@ struct drgn_error *linux_kernel_object_find(const char *name, size_t name_len,
 			if (flags & DRGN_FIND_OBJECT_CONSTANT)
 				return linux_kernel_get_vmemmap(prog, ret);
 			break;
+		@case "VMCOREINFO"@
```

Comment:
Nit: these are sorted lexicographically, could you please move this to after `UTS_RELEASE`?